auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java

```diff
@@ -1,9 +1,28 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021 Red Hat, Inc., and individual contributors
```

Comment:
Just a very small nit, this should be 2022 :)

Reply:
Thanks for the catch! Updated.